### DISCUSSION TITLE
Representing CHA₂DS₂-VASc Score in the Synthea Module Builder

### DISCUSSION THREAD

**Discussion by tamasVarjas** [synthetichealth/synthea#1057] [Q&A] [answered]
Currently, I have a task which includes creating patient data fulfilling a pre-defined set of criteria (to be used by an application), which includes having a [CHA₂DS₂-VASc Score](https://www.mdcalc.com/cha2ds2-vasc-score-atrial-fibrillation-stroke-risk) higher than a specified value. My question regarding this is twofold.

First, I am not completely sure about the category of Observation to be used in this case, although having studied the description on the relevant [HL7 site](https://www.hl7.org/fhir/valueset-observation-category.html#expansion), I think _survey_ would be the most appropriate.

Second, and perhaps more importantly, I have been provided a SNOMED code ([713678009](https://bioportal.bioontology.org/ontologies/SNOMEDCT?p=classes&conceptid=713678009)) that is being used by the application to check for this specific criteria (therefore, I am required to use this), however, the Synthea Module Builder will only allow the use of LOINC codes in Observations – which, as far as I am aware, does not exist for this specific concept, the CHA₂DS₂-VASc Score. Is it possible to specify a different code system (SNOMED) to be used here, other than LOINC? Or would I need to somehow post-process the JSON bundle created by Synthea?

![image](https://user-images.githubusercontent.com/33121266/167100464-cc89bfc7-d1c1-4fa2-881f-f61b60cd4cc9.png)

**Comment by jawalonoski** [comment, marked as answer]
I agree, the category of `survey` is probably most appropriate.

If you hand edit the system to `SNOMED-CT` it should work. The Module Builder editor does not allow this, but the Synthea engine does. So, modify your module by hand.

e.g., here is the JSON representation of your image (abridged)
```json
      "type": "Observation",
      "category": "survey",
      "unit": "",
      "codes": [
        {
          "system": "LOINC",
          "code": 713678009,
          "display": "CHAD2DS2-VASc score"
        }
      ],
```

Change the system to be as below, and the engine should export it as SNOMED.

```json
      "type": "Observation",
      "category": "survey",
      "unit": "",
      "codes": [
        {
          "system": "SNOMED-CT",
          "code": 713678009,
          "display": "CHAD2DS2-VASc score"
        }
      ],
```
